tools/Resources.py

```diff
@@ -0,0 +1,60 @@
+import os
+import requests
+from urllib3 import disable_warnings, exceptions
+from requests.auth import HTTPBasicAuth
+
+
+class Resources:
+
+    def get_resources(self, target, **kwargs):
+        url = "https://" + target + "/suite-api/api/resources"
+        querystring = {
+            'parentId': kwargs['parentid'],
+            'adapterKind': 'VMware',
+            'resourceKind': kwargs['resourcekind'],
+            'pageSize': '50000'
+        }
+        headers = {
+            'Content-Type': "application/json",
+            'Accept': "application/json"
+        }
+        disable_warnings(exceptions.InsecureRequestWarning)
+        resources = list()
+        response = requests.get(url,
```

Comment:
Would put this in a try except case, calls can go wrong in many ways tho

Reply:
resolved